res/DegenEmbed.py

```diff
@@ -0,0 +1,23 @@
+import discord
+
+
+class DegenEmbed:
+    def __init__(self, title, description, logo=None, color=0x1f6e9e):
+        self.embed_title = title
+        self.embed_description = description
+        self.embed_logo = logo
+        self.embed_color = color
+        self.embed = discord.Embed(title=self.embed_title, url="https://www.seattledegens.wtf", description=self.embed_description,
+                                   color=self.embed_color)
+
+# Need to_dict for some embed internals.
+    def to_dict(self):
+        return self.embed.to_dict()
+
+    def create(self):
```

Comment:
don't quite understand the purpose of this method. Shouldn't this be like a one-time class? Why make a create method?

Reply:
This is because of the way embeds are injected. without this it fails.